locator/tablets.hh

```diff
@@ -302,6 +336,15 @@ struct load_stats {
     }
 };
 
+struct repair_scheduler_config {
+    // When set to false, auto repair is disabled
+    bool auto_repair_enabled = false;
+    // If the time since last repair is bigger than auto_repair_threshold
+    // seconds, the tablet is eligible for auto repair.
+    std::chrono::seconds auto_repair_threshold{10 * 24 * 3600};
```

Comment:
I suggest to include the units in the member name, and also change threshold to interval as we use in other places. resulting in `auto_repair_interval_in_seconds`

Reply:
The member has a type std::chrono::seconds which already indicates the unit is seconds.  Technically it is not an interval, because it is quite possible a tablet will not be repaired when the "interval" has elapsed due to scheduling constrains. The variable only gives a hint when a tablet is eligible to be repaired.

Reply:
We use this member auto_repair_threshold in the code. We do not need to know if it is seconds or not in the code to compare the time since it is std::chrono::seconds. If we used the type int for auto_repair_threshold. It makes more sense to use the name auto_repair_threshold_in_seconds.

Reply:
Yes, repeating the unit does not add anything.